octoprint_mrbeam/iobeam/lid_handler.py

```diff
@@ -1371,10 +1391,10 @@ def save_camera_settings(
         settings = {}
         try:
             with open(path) as f:
-                settings = yaml.load(f)
-        except (OSError, IOError) as e:
+                settings = yaml.safe_load(f)
+        except (OSError, IOError, yaml.YAMLError, yaml.reader.ReaderError, AttributeError) as e:
             self._logger.warning(
```

Comment:
This should be `self._logger.warn`
I'm not sure if it works with `warning` too 
I know it's not part of the changes but just wanted to mention
It's used in many locations in our plugin but in OctoPrint it is always: `self._logger.warn`

Reply:
we have warning method in Mr beam plugin